src/components/AlertBanner.js

```diff
@@ -0,0 +1,38 @@
+import { makeStyles } from "@material-ui/core/styles";
+import Alert from "@material-ui/lab/Alert";
+import AlertTitle from "@material-ui/lab/AlertTitle";
+
+const useStyles = makeStyles((theme) => ({
+    alertBanner: {
+        marginBottom: theme.spacing(2),
+        display: "flex",
+    },
+}));
+
+const alerts = [
+    /*    {
+        startDate: "2021-03-04T06:00:00-05:00",
+        endDate: "2021-03-04T09:00:00-05:00",
+        severity: "warning",
```

Comment:
If we anticipate this being editing in a rush by inexperienced React developers, perhaps a coment summarizing the choices for severity, which seem to be: success, info, warning, error.